generators/dart.js

```diff
@@ -198,3 +205,63 @@ Blockly.Dart.scrub_ = function(block, code) {
   var nextCode = Blockly.Dart.blockToCode(nextBlock);
   return commentCode + code + nextCode;
 };
+
+/**
+ * Gets a property and adjusts the value (taking into account indexing).
+ * @param {Blockly.Block} block the block
```

Comment:
!Blockly.Block since the block may not be null.
Also, all descriptors (in this case "the block") should start with an capital letter and end with punctation (except if quoting code or something).